pandas/core/missing.py

```diff
@@ -477,6 +496,65 @@ def _akima_interpolate(xi, yi, x, der=0, axis=0):
     else:
         return [P(x, nu) for nu in der]
 
+def interpolate_1d_fill(
+    values,
+    method="pad",
+    axis=0,
+    limit=None,
+    limit_area=None,
+    fill_value=None,
+    dtype=None,
```

Comment:
```suggestion
    dtype: Optional[Dtype] = None,
```

(import from pandas._typing)